2.6.1

MaxUlysse opened the pull request:
# nf-core/sarek pull request

Many thanks for contributing to nf-core/sarek!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/sarek branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/sarek)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)

ggabernet commented on .github/workflows/awsfulltest.yml:
```diff
@@ -0,0 +1,40 @@
+name: nf-core AWS test
```
```suggestion
name: nf-core AWS full test
```

ggabernet commented on .github/workflows/awsfulltest.yml:
```diff
@@ -0,0 +1,40 @@
+name: nf-core AWS test
+# This workflow is triggered on push to the master branch and on published releases.
+# It runs the -profile 'test' on AWS batch
```
```suggestion
# It runs the -profile 'test_full' on AWS batch
```

apeltzer commented:
Yeah the PR comes from your branch so the branch checks fail for a reason, shouldn't be bad ;-)

ggabernet commented:
In [line 2709](https://github.com/nf-core/sarek/blob/c6a4e9d7074d40ea26e763e5a22eaba38fa32120/main.nf#L2709) of `main.nf`  : 
The indentation is weird.

And in line [2710](https://github.com/nf-core/sarek/blob/c6a4e9d7074d40ea26e763e5a22eaba38fa32120/main.nf#L2710) of `main.nf` a channel is created that I think it is not used afterward (or I could not find it).

ewels commented on CHANGELOG.md:
```diff
@@ -4,7 +4,33 @@ All notable changes to this project will be documented in this file.
+### Changed
+
+- [#208](https://github.com/nf-core/sarek/pull/208) - Merge changes from the release PR
+- [#208](https://github.com/nf-core/sarek/pull/208) - Bump version to `3.0dev`
```
2.6.1, not 3.0..

MaxUlysse replied:
Actually that PR did bump to 3.0dev. I'm just changing it out to 2.6.1 in this PR.

MaxUlysse commented:
> In [line 2709](https://github.com/nf-core/sarek/blob/c6a4e9d7074d40ea26e763e5a22eaba38fa32120/main.nf#L2709) of `main.nf` :
> The indentation is weird.
> 
> And in line [2710](https://github.com/nf-core/sarek/blob/c6a4e9d7074d40ea26e763e5a22eaba38fa32120/main.nf#L2710) of `main.nf` a channel is created that I think it is not used afterward (or I could not find it).

Fixed your comments with #223 that was just merged.
And I updated this branch.

So I think we're good to go ;-)

ggabernet commented:
ok, great! Is it normal that one of the CI tests (annotation) fails?

MaxUlysse commented:
> ok, great! Is it normal that one of the CI tests (annotation) fails?

I don't know why the circleci was not triggered on this PR.
Maybe it's because it's coming from my fork.

But basically the images for the annotation are not built, but they will be built once the PR is merged.

Worst case scenario, I can still build them all myself.

I can actually build the one for the tests now if you like.

ggabernet commented:
All good, whatever works best for you :)
